Assets/GothicVR/Scripts/Importer/PhoenixImporter.cs

```diff
@@ -116,9 +117,16 @@ private void LoadSfxVM(string G1Dir)
             PhoenixBridge.VmSfxPtr = vmPtr;
         }
 
-        private void LoadTestMusic(string G1Dir)
+        private void LoadMusicVM(string G1Dir)
         {
-            SingletonBehaviour<MusicTest>.GetOrCreate().Create(G1Dir);
+            var fullPath = Path.GetFullPath(Path.Join(G1Dir, "/_work/DATA/scripts/_compiled/MUSIC.DAT"));
+            var vmPtr = VmGothicBridge.LoadVm(fullPath);
+            PhoenixBridge.VmMusicPtr = vmPtr;
+        }
+
+        private void LoadMusic(string G1Dir)
+        {
+            SingletonBehaviour<MusicCreator>.GetOrCreate().Create(G1Dir);
```

Comment:
What is it doing? I see you have .Create() method in here but don't return anything. To be aligned with the other Creator.cs classes, please return the value you loaded and store it somewhere.
example: _MeshCreator_ or _VobCreator_

Reply:
Moved music initialization logic from Create to Start to so we do not need this function anymore. Music will automatically be created and checking for the debug flag to play the music or not